ci/README.md

```diff
@@ -0,0 +1,60 @@
+# Continuous Integration
+
+Every pull request in the docs repository is automatically built, deployed to a
+test S3 bucket (`cockroach-docs-review`), and run through HTML proofer. Only if
+all three of these steps succeed does the PR get a green checkmark from CI.
+
+This process is orchestrated by the `pull-request` script in this directory.
+*Please* do not make changes to the build process in the CI build configuration.
+Versioning the build script along with the code means we can isolate build
```

Comment:
Nope, just the one `ci` directory! What I mean is that it'll be possible to make changes to the build system without breaking all other PRs, because the build system configuration lives in this repository. (E.g., when I switched us to redcarpet, I fixed the build configuration for my branch but broke all other branches in the process.)